File: LeetCode Solutions/November LeetCoding Challenge/Minimum Cost to Move Chips to The Same Position.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int oddNumbers = 0;
        int evenNumbers = 0;
        for(int i : position){
            if(i%2 == 0){
                evenNumbers += 1;
            } else{
                oddNumbers += 1;
            }
        }
        if(evenNumbers > oddNumbers){
            return oddNumbers;
        }
        return evenNumbers;
    }
};

File: LeetCode Solutions/November LeetCoding Challenge/Minimum Cost to Move Chips to The Same Position.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        oddNumbers, evenNumbers = 0, 0
        for i in position:
            if(i%2 == 0):
                evenNumbers += 1
            else:
                oddNumbers += 1
        if(evenNumbers > oddNumbers):
            return oddNumbers
        return evenNumbers